portfolio/src/main/webapp/index.html

```diff
@@ -47,7 +47,7 @@ <h2>Background</h2>
       <br>
       <h2>Projects:</h2>
       <div id="projects">
-        <a href="https://github.com/amuamushu/cogs9_final_project" target="_blank">
+        <a href="https://drive.google.com/file/d/1Jb52Jk0HLoNbdVDg3z5VrQMrCsxFrkv-/view?usp=sharing" target="_blank">
```

Comment:
Is this your personal drive? I am assuming yes.

Reply:
Yes, it is a link to a pdf in my personal drive. Should I move the file to my @google drive or remove it completely?

Reply:
No, because once you leave you want this to still work :)

Reply:
Understood! 
Done